python/dgl/function/base.py

```diff
@@ -15,9 +15,9 @@ class TargetCode(object):
     EDGE = 2
 
     CODE2STR = {
-        0: "src",
-        1: "dst",
-        2: "edge",
+        0: "u",
```

Comment:
Does this mean we no longer support `src_mul_edge`?

Reply:
We still do. `src_mul_edge` is just an alias for `u_mul_e`.